models/issues/issue_project.go

```diff
@@ -90,22 +90,16 @@ func LoadIssuesFromBoardList(ctx context.Context, bs project_model.BoardList) (m
 	return issuesMap, nil
 }
 
-// ChangeProjectAssign changes the project associated with an issue
-func ChangeProjectAssign(ctx context.Context, issue *Issue, doer *user_model.User, newProjectID int64) error {
-	ctx, committer, err := db.TxContext(ctx)
-	if err != nil {
-		return err
-	}
-	defer committer.Close()
-
-	if err := addUpdateIssueProject(ctx, issue, doer, newProjectID); err != nil {
-		return err
-	}
-
-	return committer.Commit()
+// ChangeProjectAssign changes the project associated with an issue, if newProjectID is 0, the issue is removed from the project
+func ChangeProjectAssign(ctx context.Context, issue *Issue, doer *user_model.User, newProjectID, newColumnID int64) error {
```

Comment:
ChangeProjectAssign is exactly addUpdateIssueProject? Why it needs this wrapper

Reply:
cf9ef105101fbc13f1e971b45036a6a46edf5b19
functions renamed.